theme/templates/accounts/profile.html

```diff
@@ -358,6 +358,73 @@ <h5><span class="text-muted">Date: </span>{{ res.created }}</h5>
+                                                            </div>
+                                                        </div>
+                                                        <span class="text-danger">You are over your allocated quota by {{ qd.percent_over|floatformat:"1" }}%</span>
+                                                    {% else %}
+                                                        <div class="progress">
+                                                            <div class="progress-bar progress-bar-success" 
+                                                                style="width: {{ qd.uz_percent }}%" 
+                                                                title="{{ qd.uz_percent|floatformat:"1" }}% used by userZone">
+                                                                <span class="sr-only">{{ qd.uz_percent|floatformat:"1" }}% of quota used by userZone</span>
+                                                            </div>
+                                                            <div class="progress-bar progress-bar-info" 
+                                                                style="width: {{ qd.dz_percent }}%"
+                                                                title="{{ qd.dz_percent|floatformat:"1" }}% used by dataZone">
+                                                                <span class="sr-only">{{ qd.dz_percent|floatformat:"1" }}% of quota is used by dataZone</span>
+                                                            </div>
+                                                        </div>
+                                                        <span>{{ qd.remaining }}{{ qd.unit }} free of {{ qd.allocated }}{{ qd.unit }}</span>
+                                                    {% endif %}
+                                                </div>
+                                                {% endwith %}
+                                                {% if quota_message %}
+                                                    <div class="alert alert-info">
+                                                        {{ quota_message }}
+                                                        Additional Quota information can be found <a href="https://help.hydroshare.org/about-hydroshare/policies/quota/" target="_blank">here.</a>
+                                                    </div>
+                                                {% endif %}
+                                                {% if not quota_requests %}
+                                                    <button id="quota-request-storage" type="button" data-toggle="modal" data-target="#manage-quota-dialog" class="btn btn-primary has-space-bottom">
+                                                        <i class="fa fa-plus" aria-hidden="true"></i> Request additional storage
```

Comment:
I think this button is too prominent for a function that won't be used too frequently. I think it should use the `default` button style.

Reply:
sounds good!